[2020-06-18 09:21:54,096] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:00:00+00:00 [queued]>
[2020-06-18 09:21:54,103] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:00:00+00:00 [queued]>
[2020-06-18 09:21:54,103] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 09:21:54,104] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-18 09:21:54,104] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 09:21:54,113] {taskinstance.py:900} INFO - Executing <Task(SlackWebhookOperator): slack_msg> on 2018-08-11T00:00:00+00:00
[2020-06-18 09:21:54,115] {standard_task_runner.py:53} INFO - Started process 10023 to run task
[2020-06-18 09:21:54,176] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: slack_example.slack_msg 2018-08-11T00:00:00+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-18 09:21:54,198] {taskinstance.py:1145} ERROR - Cannot get token: No valid Slack webhook token nor conn_id supplied
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/lib/python2.7/site-packages/airflow/contrib/operators/slack_webhook_operator.py", line 111, in execute
    self.proxy
  File "/usr/lib/python2.7/site-packages/airflow/contrib/hooks/slack_webhook_hook.py", line 79, in __init__
    self.webhook_token = self._get_token(webhook_token, http_conn_id)
  File "/usr/lib/python2.7/site-packages/airflow/contrib/hooks/slack_webhook_hook.py", line 108, in _get_token
    raise AirflowException('Cannot get token: No valid Slack '
AirflowException: Cannot get token: No valid Slack webhook token nor conn_id supplied
[2020-06-18 09:21:54,202] {taskinstance.py:1202} INFO - Marking task as FAILED.dag_id=slack_example, task_id=slack_msg, execution_date=20180811T000000, start_date=20200618T092154, end_date=20200618T092154
[2020-06-18 09:22:04,087] {logging_mixin.py:112} INFO - [2020-06-18 09:22:04,087] {local_task_job.py:103} INFO - Task exited with return code 1
[2020-06-18 10:38:53,722] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:00:00+00:00 [queued]>
[2020-06-18 10:38:53,730] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: slack_example.slack_msg 2018-08-11T00:00:00+00:00 [queued]>
[2020-06-18 10:38:53,730] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 10:38:53,730] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-18 10:38:53,730] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 10:38:53,739] {taskinstance.py:900} INFO - Executing <Task(SlackWebhookOperator): slack_msg> on 2018-08-11T00:00:00+00:00
[2020-06-18 10:38:53,741] {standard_task_runner.py:53} INFO - Started process 4245 to run task
[2020-06-18 10:38:53,800] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: slack_example.slack_msg 2018-08-11T00:00:00+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-18 10:38:53,820] {logging_mixin.py:112} INFO - [2020-06-18 10:38:53,820] {base_hook.py:87} INFO - Using connection to: id: anitha_slack. Host: https://hooks.slack.com/services, Port: None, Schema: None, Login: None, Password: XXXXXXXX, extra: None
[2020-06-18 10:38:53,830] {logging_mixin.py:112} INFO - [2020-06-18 10:38:53,829] {base_hook.py:87} INFO - Using connection to: id: anitha_slack. Host: https://hooks.slack.com/services, Port: None, Schema: None, Login: None, Password: XXXXXXXX, extra: None
[2020-06-18 10:38:53,835] {logging_mixin.py:112} INFO - [2020-06-18 10:38:53,835] {http_hook.py:136} INFO - Sending 'POST' to url: https://hooks.slack.com/services
[2020-06-18 10:38:53,857] {logging_mixin.py:112} WARNING - /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
[2020-06-18 10:38:53,902] {logging_mixin.py:112} WARNING - /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
[2020-06-18 10:38:53,961] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=slack_example, task_id=slack_msg, execution_date=20180811T000000, start_date=20200618T103853, end_date=20200618T103853
[2020-06-18 10:39:03,714] {logging_mixin.py:112} INFO - [2020-06-18 10:39:03,714] {local_task_job.py:103} INFO - Task exited with return code 0
